cmd/rootlesskit/main.go

```diff
@@ -29,17 +30,24 @@ import (
 	"github.com/rootless-containers/rootlesskit/v2/pkg/version"
 )
 
+
+const (
+		pipeFDEnvKey              = "_ROOTLESSKIT_PIPEFD_UNDOCUMENTED"
+		childUseActivationEnvKey  = "_ROOTLESSKIT_CHILD_USE_ACTIVATION_UNDOCUMENTED"
+		runActivationEnvKey       = "_ROOTLESSKIT_RUN_ACTIVATION_UNDOCUMENTED"
```

Comment:
Probably these ones should be like `_ROOTLESSKIT_SYSTEMD_ACTIVATION_%s_UNDOCUMENTED`

Reply:
changed to _ROOTLESSKIT_SYSTEMD_ACTIVATION_*_UNDOCUMENTED